derby.log

```diff
@@ -0,0 +1,124 @@
+----------------------------------------------------------------
+Thu Dec 01 11:34:20 CET 2022:
+Booting Derby version The Apache Software Foundation - Apache Derby - 10.9.1.0 - (1344872): instance a816c00e-0184-cd40-5dc4-00003b6fd290 
```

Comment:
I think derby.log should be put on .gitignore

Reply:
Done!